.ci_support/linux_64_python3.7.____cpython.yaml

```diff
@@ -14,8 +14,6 @@ cxx_compiler_version:
 - '9'
 docker_image:
 - quay.io/condaforge/linux-anvil-comp7
-numpy:
-- '1.17'
```

Comment:
The fact these are going away bothers me. Hopefully it is still building the minimum NumPy.

Reply:
Had a suggestion that might fix this this issue